locust/input_events.py

```diff
@@ -0,0 +1,91 @@
+import time
+
+isWindows = False
```

Comment:
This is a nicer way to know if we're on Windows (no need to try/catch & no need to store our own variable)

```
import os

if os.name == 'nt':
     ...
```